tests/src/main.F90

```diff
@@ -934,7 +935,8 @@ program main
      !> Loading the tb parameters (electrons.dat)
      call load_latteTBparams(tbparams,mol%splist,"./")
      call write_latteTBparams(tbparams,"myelectrons.dat")
-     call system("diff -qs  myelectrons.dat electrons.dat > tmp.tmp")
+     call system("diff -qs --ignore-all-space myelectrons.dat electrons.dat > tmp.tmp")
+     call system("diff  --ignore-all-space myelectrons.dat electrons.dat")
```

Comment:
If you would like to print the `diff` _and_ write it to file you can do this in one line with
```
call system("diff --report-identical-files --ignore-all-space myelectrons.dat electrons.dat | tee tmp.tmp")
```

Reply:
I have expanded the `-s` option to `--report-identical-files` which I personally find more readable. But that's not necessary for the above to work.